readthedocs/projects/tasks/search.py

```diff
@@ -109,6 +109,31 @@ def reindex_version(version_id, search_index_name=None):
         log.exception("Failed during creation of new files")
 
 
+@app.task(queue="web")
+def index_project(project_slug, skip_if_exists=False):
+    """
+    Index all active versions of the project.
+
+    If ``skip_if_exists`` is True, we first check if
+    the project has at least one version indexed,
+    and skip the re-indexing if it does.
+    """
+    log.bind(project_slug=project_slug)
+    project = Project.objects.filter(slug=project_slug).first()
+    if not project:
+        log.debug("Project doesn't exist.")
+        return
+
+    if skip_if_exists:
+        if PageDocument().search().filter("term", project=project.slug).count():
```

Comment:
```suggestion
        if PageDocument().search().filter("term", project=project.slug).exists():
```

I'm not sure if `.exists()` works here, but I'm suggesting it since we don't really need the count.

Reply:
ES doesn't have an exists endpoint.